docs/columns.md

```diff
@@ -419,6 +421,35 @@ A new `Object` will be the result of element headerAttrs.
 > Same as [column.attrs](#attrs), it has lower priority and will be
 > overwrited when other props related to HTML attributes were given.
 
+### <a name='headerSortingClasses'>headerSortingClasses - [String | Function]</a>
+
+`headerSortingClasses` allows to customize `class` for header cell which the table was sorting based on. 
```

Comment:
`headerSortingClasses` allows to customize `class` for header cell when this column is sorting